core/kazoo_number_manager/src/carriers/knm_bandwidth2.erl

```diff
@@ -54,33 +55,48 @@
 
 -ifdef(TEST).
 -define(BW2_ACCOUNT_ID, "eunit_testing_account").
+-define(IS_SANDBOX_PROVISIONING_TRUE, 'true').
```

Comment:
Oh well, certainly can't change things that have been that way for over a year, like tickies on atoms, and...